db/schema.rb

```diff
@@ -1283,6 +1282,7 @@
     t.datetime "last_reacted_at"
     t.datetime "last_sign_in_at"
     t.inet "last_sign_in_ip"
+    t.datetime "latest_article_updated_at"
```

Comment:
This is just reordered, probably due to the recent switch from fix-db-schema-conflicts to strong-migrations.